### PR TITLE
Update the new chapter checklist re: meetup fees

### PR DESCRIPTION
**Pull request by etownbetty**
We updated the funding info for PSF

Closes https://github.com/pyladies/pyladies-kit/issues/48

### CODE REVIEW

File: docs/prospective/index.rst [hunk 1]
```diff
@@ -91,11 +91,14 @@ Once you have received a confirmation that the the initial PyLadies request form
...
+3. If your Meetup groups' 6-month subscription is $89.94 USD (or equivalent) or more, please contact accounting@python.org to request being added to the PSF Meetup Pro network and include a link to your group's code of conduct.
+
+   Meetup subscription reimbursement requests should be submitted within 120 days from the start of a meetup subscription period or the reimbursement will not be approved. For example, if a meetup subscription period begins on June 25, 2018, the organizer may submit a grant request before June 25, 2018 or up to October 23, 2018.
 
 .. note::
 
-    The Python Software Foundation will reimburse Meetup.com fees for PyLadies Meetup groups.  However, if your location does not use Meetup.com but a different/similar site that charges, **still file a reimbursement!**
+    Reimbursements of meetup subscriptions made to PSF fiscal sponsorees are not charged against the fiscal sponsoree's fund balance.
+    If you would like more detail go to the `PSF FAQ page`_
```
Comment (Mariatta) [inline docs/prospective/index.rst]:
```suggestion
    If you would like more detail go to the `PSF FAQ page`_.
```
Missing a period at the end 😅

**Comment by Mariatta** [comment]
Thanks @etownbetty!! 🎉

**Comment by Mariatta** [comment]
@econchick I've merged the PR, but after half an hour, the change still doesn't seem to be reflected at kit.pyladies.com.
Could you check perhaps in readthedocs side if there was problem with building the doc?
Thanks.

**Comment by Mariatta** [comment]
@econchick I've merged the PR, but after half an hour, the change still doesn't seem to be reflected at kit.pyladies.com.
Could you check perhaps in readthedocs side if there was problem with building the doc?
Thanks.

**Comment by econchick** [comment]
@Mariatta it looks like it didn't get triggered; the last successful build was almost two years ago (wow). It seems the webhook got deleted somehow, so I re-added it. 

The RTDs account is actually a general PyLadies account under info@pyladies.com. I can send you the password (looks like the email also has to be verified w RTDs).

**Comment by econchick** [comment]
(btw I just manually triggered a build)

**Pull request merged**